chapters/functions.tex

```diff
@@ -2243,13 +2243,15 @@ \subsection{Annotations for External Libraries and Include Files}\label{annotati
     input Real x;
     output Real y;
     external "C"
-    y=ExternalFunc1_ext(x) annotation(Library="ExternalLib1",Include="#include \"ExternalFunc1.h\"");
+    y=ExternalFunc1_ext(x) annotation(Library="ExternalLib1",
+      Include="#include \"ExternalFunc1.h\"");
   end ExternalFunc1;
 
   function ExternalFunc2
     input Real x;
     output Real y;
-    external "C" annotation(Library="ExternalLib2", Include="#include \"ExternalFunc2.h\"");
+    external "C" annotation(Library="ExternalLib2", 
+      Include="#include \"ExternalFunc2.h\"");
```

Comment:
Similarly (or is it still too long):
```suggestion
    external "C"
      annotation(Library="ExternalLib2", Include="#include \"ExternalFunc2.h\"");
```

Reply:
I think it's still too long.

Reply:
Of course, there is also the really extreme approach where the `Include` is aligned with `Library`, but to look nice we would then end up with two line breaks instead of one:
```
    external "C"
      annotation(Library="ExternalLib2",
                 Include="#include \"ExternalFunc2.h\"");
```

Anyway, these are just formatting ideas about something that clearly isn't necessary.